src/index.html

```diff
@@ -3,11 +3,393 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>Miami Condo Kings</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link
+      rel="shortcut icon"
+      href="images/favicon.png"
+      type="image/x-icon"
+    >
   </head>
   <body>
-    <h1>Miami</h1>
+    <header class="header content-container" id="home">
+      <div class="header__top-container">
+        <a href="tel:+12345555555" class="header__item--phone-icon"></a>
+        <a href="tel:+12345555555" class="header__item header__item--phone-text">
+          +1 234 555-55-55
+        </a>
+        <a href="#" class="header__item big-logo header__item--logo">
+          <img
+            src="images/logo.png"
+            alt="miami logo"
+            class="big-logo__img"
+          >
+        </a>
+        <a href="#" class="header__item header__item--underlined">
+          call to order
+        </a>
+        <label for="menuOpener" class="header__item--menu-button">
+        </label>
+      </div>
+      <section class="header__bottom-container">
+        <h1 class="header__title">
+          Miami<br>Condo Kings
+        </h1>
+        <nav>
+          <ul class="header__links-container">
+            <li>
+              <a href="#home" class="header__item header__item--active">home</a>
+            </li>
+            <li>
+              <a href="#aboutUs" class="header__item">about us</a>
```

Comment:
```suggestion
              <a href="#aboutUs" class="header__item">About us</a>
```
deafult -> first uppercase letter

Reply:
а чого так?

Reply:
because if you decide to use usual style (lowercase letters with uppercase first letter), you won't need to change the whole document

uppercase and lowercase styles are always available in css

Reply:
ok, done